server/pull_request_handler.go

```diff
@@ -96,11 +96,6 @@ func (s *Server) pullRequestEventHandler(w http.ResponseWriter, r *http.Request)
 			return
 		}
 
-		if (pr.RepoName == s.Config.E2EServerReponame || pr.RepoName == s.Config.E2EWebappReponame) && contains(s.Config.E2ETriggerLabel, *event.Label.Name) {
-			mlog.Info("Label to run e2e tests", mlog.Int("pr", event.PRNumber), mlog.String("repo", pr.RepoName), mlog.String("label", *event.Label.Name))
-			go s.triggerE2ETestFromLabel(pr)
-		}
```

Comment:
Can we also remove the label from https://github.com/mattermost/mattermost ?

Reply:
Absolutely :+1: I can do that already

Reply:
Ah wait, those are not only for triggering the test, but also for marking PRs for QA testers. The two labels that exist right now should remain as they are:
- `3: QA Review`
- `QA Deferred`

Reply:
Also for best compatibility i'd port this label-triggered functionality to GHA as well, i'll go ahead and prepare a PR for that